Assets/General/Scripts/Managers/UniversalManagers/MainUniversalManagers/AmbienceManager.cs

```diff
@@ -164,6 +164,10 @@ private void StartAmbience(EventReference eventReference)
 
     private void StartAmbienceOnObject(EventReference eventReference, GameObject audioObject)
```

Comment:
I know this wasn't in your changes to this but this should have an XML comment. Guess this was missed on a previous code review